packages/docs/src/components/Select/Select.docs.mdx

```diff
@@ -0,0 +1,106 @@
+      </Select>
+      <FormHelperText>Some important helper text</FormHelperText>
+    </FormControl>
+  );
+};
+
+```
+
+For more advanced usages, check [Material UI Select docs](https://material-ui.com/pt/components/selects/)
+
+---
+
+---
+
+## Select Props
+
+| Name               |                 Type                 |      Default      |                                                                                                                                                                                                                               Description                                                                                                                                                                                                                               |
+| :----------------- | :----------------------------------: | :---------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| autoWidth          |                 bool                 |       false       |                                                                                                                                                    If true, the width of the popover will automatically be set according to the items inside the menu, otherwise it will be at least the width of the select input.                                                                                                                                                     |
+| children           |                 node                 |                   |                                                                                                                                    The option elements to populate the select with. Can be some MenuItem when native is false and option when native is true. The MenuItem elements must be direct descendants when native is false.                                                                                                                                    |
+| classes            |                object                |                   |                                                                                                                                                                Override or extend the styles applied to the component. See [CSS API](https://material-ui.com/pt/api/select/#css) below for more details.                                                                                                                                                                |
```

Comment:
Change the `CSS API below` text to something pointing to material-ui docs for Select